code/mobile/android/PhoneVR/app/build.gradle

```diff
@@ -80,7 +78,23 @@ android {
         }
     }
     buildFeatures {
-        buildConfig = true // TODO: Remove BuildConifg, Gradle suggests this usage is inefficient > v8 disabled by default
+        buildConfig = true
+        viewBinding true
+    }
+    flavorDimensions "gvr"
+    productFlavors {
+        gvr {
+            dimension "gvr"
+            externalNativeBuild {
+                cmake {
+                    arguments "-D HAVEGVR=true"
+                }
+            }
+        }
+        noGvr {
+            dimension "gvr"
+            versionNameSuffix "-lite"
```

Comment:
I think its better to leave it as -nogvr, -lite would be misleading

Reply:
Done